src/30_Advanced/Copy_Button.html

```diff
@@ -0,0 +1,93 @@
+<!---{
+  "preview": "default",
+  "default": "preview"
+}--->
+<!-- ## Introduction -->
+<!--
+This is a sample showing how to add a "copy" button to your page, allowing the user to copy some
+fixed item per-page. This uses an `amp-iframe`, running custom JS, that loads in the background
+while a placeholder button remains visible.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+
+  <!-- ## Setup -->
+  <!-- We use `amp-iframe` to load a copy button, replacing a placeholder button.-->
+  <script async custom-element="amp-iframe" src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js"></script>
+
+  <link rel="canonical" href="<%host%>/advanced/copy_button/">
+  <title>Copy Button</title>
+
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+  <style amp-custom>
+/* common styles for button */
+button#copy{
+  box-sizing: border-box;
+  font: inherit;
+  font-size: 18px;
+  line-height: 2em;
+  cursor: pointer;
+  border-radius: 4px;
+  border: 2px solid transparent;
+  background: #555;
+  padding: 0 8px;
+  color: #fff;
+  display: block;
+}
+/* disable for placeholder */
+button#copy {
+  cursor: not-allowed;
+  width: 100%;
+  height: 100%;
+}
+/* misc page CSS */
+body {
+  font-family: Sans-Serif;
+}
+main {
```

Comment:
Can you make this rule less generic, otherwise you're messing with the overall layout.

Reply:
Done